### ISSUE TITLE
Delete test_webclient.py's testPartial() test, which is being skipped

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8530]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8530|
|Type|defect|
|Created|2016-06-28 09:04:06Z|
|Branch|https://github.com/twisted/twisted/tree/8530-rodrigc-web|

The testPartial() method in twisted/web/test/test_webclient.py
is not receiving code coverage during test runs:
https://codecov.io/gh/twisted/twisted/src/f2ce17f06c9f935f9ff9ec3f1487c9dda3ff37fa/twisted/web/test/test_webclient.py#L691

It was marked as a skipped test in 2004.

<details><summary>Searchable metadata</summary>

```
trac-id__8530 8530
type__defect defect
reporter__rodrigc rodrigc
priority__normal normal
milestone__None None
branch__8530_rodrigc_web 8530-rodrigc-web
branch_author__ 
status__closed closed
resolution__fixed fixed
component__web web
keywords__None None
time__1467104646562599 1467104646562599
changetime__1467138600718941 1467138600718941
version__None None
owner__rodrigc rodrigc

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/254

**Comment by twisted-trac** [comment]
|[<img alt="Lukasa's avatar" src="https://avatars.githubusercontent.com/u/1382556?s=50" width="50" height="50">](https://github.com/Lukasa)<a name="note_4"></a>|@Lukasa set owner to @rodrigc|
|-|-|

Windows test failure is irrelevant. LGTM, merge when ready.

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_5"></a>|GitHub set status to `closed`|
|-|-|

In changeset cd44b8b799551e633240d2fbd850adbf7868d12a
```
#!CommitTicketReference repository="" revision="cd44b8b799551e633240d2fbd850adbf7868d12a"
Merge 8530-rodrigc-web: Delete testPartial() from test_webclient.py, which has been skipped since 2004

Author: rodrigc
Reviewer: Lukasa
Fixes: #8530
```
